src/Chapter2.hs

```diff
@@ -740,7 +751,8 @@ the list with only those lists that contain a passed element.
 
 🕯 HINT: Use the 'elem' function to check whether an element belongs to a list
 -}
-contains = error "contains: Not implemented!"
+contains :: Int -> [[Int]] -> [[Int]]
+contains num l = filter (\x -> elem num x) l
```

Comment:
👌🏼 
You Could use eta-reduction both in the function itself and in the lambda function 🙂